Common/include/matrix_structure.hpp

```diff
@@ -239,6 +239,29 @@ class CSysMatrix {
 	 */
 	void DeleteValsRowi(unsigned long i);
   
+  /*!
+   * \brief Recursive definition of determinate using expansion by minors. Written by Paul Bourke
```

Comment:
Looking at Paul Bourke web page, he says "Any source code found here may be freely used provided credits are given to the author." so we are OK. Anyway, these are standard algorithms that we can find in any algebra book.